pod/video/management/commands/bbb.py

```diff
@@ -18,7 +18,8 @@
 
 - Search to matching BBB users as Pod users.
 This allows to try if BBB user (known with firstname and lastname) is matching
-a Pod user.
+a Pod user. You can parameter the BBB username format via the use
+of BBB_USERNAME_FORMAT setting.
```

Comment:
@LoicBonavent  a pod user. You can set the BBB username format through the use
of the BBB_USERNAME_FORMAT setting.